system/Log/Logger.php

```diff
@@ -252,13 +252,8 @@ public function log($level, $message, array $context = []): bool
             $level = array_search((int) $level, $this->logLevels, true);
         }
 
-        // Is the level a valid level?
-        if (! array_key_exists($level, $this->logLevels)) {
-            throw LogException::forInvalidLogLevel($level);
-        }
```

Comment:
Wouldn't removing this check be a violation of the PSR Log?

> A ninth method, log, accepts a log level as the first argument. Calling this method with one of the log level constants MUST have the same result as calling the level-specific method. Calling this method with a level not defined by this specification MUST throw a Psr\Log\InvalidArgumentException if the implementation does not know about the level.
https://www.php-fig.org/psr/psr-3/

Reply:
It states 

if the implementation does not know about the level. Users SHOULD NOT use a custom level without knowing for sure the current implementation supports it.

I'll update the code to include this but it should be handled a bit differently